### PR TITLE
Feature/4497 5521 implement sending screen

### PR DESCRIPTION
**Pull request by 30mar**
## Description
Implement the confirmation screen UI, from which the user confirms to send the results.

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-5521

## Screenshots
Simulator Screen Shot - iPhone 12 - 2021-03-04 at 14 37 54

Simulator Screen Shot - iPhone 12 - 2021-03-04 at 14 37 57

### CODE REVIEW

**Review by tcfos — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/ErrorReporting/SendErrorLogsViewController.swift [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+		view.backgroundColor = .enaColor(for: .background)
+
+		dynamicTableViewModel = viewModel.sendErrorLogsDynamicViewModel
+	}
+	
+	override func viewDidAppear(_ animated: Bool) {
+		super.viewDidAppear(animated)
+		
+	}
```
Comment (whiskey) [inline src/xcode/ENA/ENA/Source/Scenes/ErrorReporting/SendErrorLogsViewController.swift:39]: redundant
```suggestion
```

**Review by whiskey — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/ErrorReporting/SendErrorLogsViewController.swift [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+		case link = "linkCell"
+		case legalExtended = "DynamicLegalExtendedCell"
+	}
+
+	// MARK: - Private
+	
+	private func setupTableView() {
+		tableView.separatorStyle = .none
+		tableView.register(
+			UINib(nibName: "ExposureDetectionLinkCell", bundle: nil),
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Scenes/ErrorReporting/SendErrorLogsViewController.swift:60]: I would prefer String(describing: ExposureDetectionLinkCell.self)

**Review by naveeddotio — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/ErrorReporting/SendErrorLogsViewController.swift (hunk 1 above)
Comment (30mar) [reply src/xcode/ENA/ENA/Source/Scenes/ErrorReporting/SendErrorLogsViewController.swift:39]: forgot to remove it, thanks

File: src/xcode/ENA/ENA/Source/Scenes/ErrorReporting/SendErrorLogsViewModel.swift [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+}
+
+extension DynamicCell {
+
+	/// A `DynamicLegalExtendedCell` to display legal text
+	/// - Parameters:
+	///   - title: The title/header for the legal foo.
+	///   - subheadline1: Optional description text.
+	///   - bulletPoints1: A list of strings to be prefixed with bullet points.
+	///   - subheadline: Optional description text.
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Scenes/ErrorReporting/SendErrorLogsViewModel.swift:96]: subheadline2

**Pull request merged**